### ISSUE TITLE
good reason for building on every install?

### ISSUE THREAD

**Issue by micahstubbs** [emeeks/semiotic#221]
curious, is there a good reason that we build the js bundle every time a dev installs js dependencies with 

commands like `yarn` or `npm install`?

```
➜  semiotic git:(micah/220/dev-dependencies) ✗ yarn
yarn install v1.6.0
[1/4] 🔍  Resolving packages...
success Already up-to-date.
$ rollup -c && uglifyjs ./dist/semiotic.js -o ./dist/semiotic.min.js && rm ./dist/semiotic.js

src/components/index.js → dist/semiotic.js...
(!) Mixing named and default exports
Consumers of your bundle will have to use bundle['default'] to access the default export, which may not be what you want. Use `exports: 'named'` to disable this warning
(!) `this` has been rewritten to `undefined`
https://github.com/rollup/rollup/wiki/Troubleshooting#this-is-undefined
src/components/DownloadButton.js
1: var _this = this;
               ^
2:
3: import React from "react";
created dist/semiotic.js in 10.5s
✨  Done in 12.84s.
```

**Comment by micahstubbs** [comment]
I ask, because this is an opportunity to remove 10.5 seconds of friction each time a new dev tries to contribute to semiotic 😉 

 if there is not a good reason for building at this command, let's remove the friction 🔓

**Comment by emeeks** [comment]
I used the default settings for everythin

**Comment by emeeks** [comment]
I think it had something to do with how `npm i` dealt with pre-publish scripts, I vaguely remember something like that changing in 4 or 5. I don't think it's necessary.

**Comment by micahstubbs** [comment]
fixed in 1ce888303b18625143b6ce6fb7bbc75649829048

thanks @rgbkrk 😄 🙌  great minds...
